custom-graders/v2/DemoJavaGrader/executeGrader.sh

```diff
@@ -0,0 +1,68 @@
+#! /bin/bash
+
+# Coursera deletes all environment variables set inside 'Dockerfile'. If any environment varables
+# need to be set, they must be set inside a wrapper bash script.
+
+local=$LOCAL
+echo $local
+
+if [ $local != "1" ]
+then
+	echo "exporting JAVA_HOME"
+	export JAVA_HOME=/usr/lib/jvm/java-7-openjdk-amd64
+fi
+
+# Switch to the grader directory
+# if [ $local != "1" ]
+# then
+#    cd /grader
+# else
+#    cd ./grader
+# fi
+
```

Comment:
Do we want to use PrimeGrader as well to illustrate examples of multiple problems/parts being authored as part of one grader? This is pretty common for partners as they like keeping everything in one grader (less effort to upload manually part-by-part for both instructors and learners). It can also clarify how we handle multiple part programming assignments contained in one image. Not sure if this is being demonstrated with the if else statement below this comment.

Reply:
Looks like. I messed up this PR. I wanted this Readme to be a separate one. I will incorporate your Readme comments into another PR.

Regarding PrimeGrader, Python Demo Grader will demonstrate the idea of using two parts. If I get more time, I will add PrimeGrader into this or create another sample.